### DISCUSSION TITLE
(not) Building fdpp with ubuntu with oibaf repo enabled

### DISCUSSION THREAD

**Discussion by stuaxo** [dosemu2/dosemu2#2001] [General]
This is not a proper bug report since I need more info, and it's a bit tricky for me to get at the moment.
I haven't run dosemu2 for a while, because I use the oibaf PPA to have up to date graphics drivers.
https://launchpad.net/~oibaf/+archive/ubuntu/graphics-drivers

Mesa uses LLVM, which comes from oibaf + this seems to lead to a conflict with clang++

At some point I need to completely uninstall the PPA and somehow revert to having none of the PPA packages installed + see how things go.

I suspect that the proper solution involves a fix in the oibaf PPA itself.

```
❯ clang++                
Command 'clang++' not found, but can be installed with:
sudo apt install clang

~/projects/external/emulation/dosemu2-projects/fdpp on master ?3 ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
❯ sudo apt install clang 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies.
 libclang-common-14-dev : Depends: libllvm14 (= 1:14.0.0-1ubuntu1) but 1:14.0.6-1~oibaf~j is to be installed
 libclang-cpp14 : Depends: libllvm14 (= 1:14.0.0-1ubuntu1) but 1:14.0.6-1~oibaf~j is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by stsp** [comment]
Its definitely not the right place to
report the clang installation problems.
Please move it to ubuntu or to PPA
or whatever.
